services/pub/pub.service.js

```diff
@@ -2,8 +2,8 @@
 
 const Joi = require('joi')
 const { BaseJsonService } = require('..')
-const { renderVersionBadge } = require('../../lib/version')
-const { latest: latestVersion } = require('../../lib/version')
+const { renderVersionBadge } = require('../version')
+const { latest: latestVersion } = require('../version')
```

Comment:
I know this was already in place, but do we really need two separate require statements for `'../version'`?

Reply:
Heh, updated!

Reply:
Lol I see you've already changed this